packages/react/src/components/InAppMessaging/BannerMessage/BannerMessage.tsx

```diff
@@ -0,0 +1,84 @@
+import isEmpty from 'lodash/isEmpty';
+
+import { Button } from '../../../primitives/Button';
+import { Flex } from '../../../primitives/Flex';
+import { Heading } from '../../../primitives/Heading';
+import { IconClose } from '../../../primitives/Icon/icons/IconClose';
+import { Image } from '../../../primitives/Image';
+import { Text } from '../../../primitives/Text';
+import { Link } from '../../../primitives/Link';
+import { useThemeBreakpoint } from '../../../hooks/useThemeBreakpoint';
+
+export default function BannerMessage({
+  alignment = 'right',
+  ...props
+}): JSX.Element | null {
+  const {
+    body,
+    header,
+    image,
+    onClose,
+    position,
+    primaryButton,
+    secondaryButton,
+  } = props;
+
+  const hasPrimaryButton = !isEmpty(primaryButton);
+  const hasSecondaryButton = !isEmpty(secondaryButton);
```

Comment:
We should be able to get these values from `useMessageProps` ([merged here](https://github.com/aws-amplify/amplify-ui/pull/2340). 

There are couple other rendering related properties (`hasRenderableImage`, `shouldRenderImage`) that we want to use from that hook, here's a reference from the React Native [BannerMessage](https://github.com/aws-amplify/amplify-ui/blob/in-app-messaging/main/packages/react-native/src/InAppMessaging/components/BannerMessage/BannerMessage.tsx#L19)